test/e2e/resources_test.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package e2e
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	placementv1beta1 "go.goms.io/fleet/apis/placement/v1beta1"
+)
+
+const (
```

Comment:
this is mostly util, is that we will add more resource related ITs in this file in the future?

Reply:
Hi Ryan! This file is mostly for utilities for building resources. IT cases will live in different files.